wifiphisher/extensions/deauth.py

```diff
@@ -181,5 +184,5 @@ def send_channels(self):
         :rtype: list
         """
         if self._is_frenzy:
-            return [str(k) for k in range(1, 14)]
+            return [str(k) for k in constants.ALL_2G_CHANNELS]
```

Comment:
ohoh! we need all the channels since we need to do the frequency hopping for all the channels : )